lib/aika_web/plugs/api_auth_plug.ex

```diff
@@ -0,0 +1,22 @@
+defmodule AikaWeb.ApiAuthPlug do
+  import Plug.Conn
+  import Phoenix.Controller
+  alias Aika.{Repo, Accounts.User}
+
+  def init(default), do: default
+
+  def call(conn, _) do
+    %{"api_token" => api_token, "user_id" => user_id} = conn.params
```

Comment:
I believe this will error if the params don't include api_token and user_id and I think either a 401 or 400 for bad request would be more appropriate than the 500 this would return